src/test/java/org/fundacionjala/virtualassistant/controllers/SpeechControllerTest.java

```diff
@@ -0,0 +1,56 @@
+package org.fundacionjala.virtualassistant.controllers;
+
+import org.fundacionjala.virtualassistant.service.SpeechService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.multipart.MultipartFile;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+public class SpeechControllerTest {
+
+    private SpeechController yourController;
```

Comment:
yourController? please change this field name to a meaningful name, avoid just coping and paste